packages/dai-plugin-governance/src/ApproveLinkTransaction.js

```diff
@@ -28,18 +28,20 @@ export default class ApproveLinkTransaction {
   }
 
   _parseLogs(logs) {
-    const { LinkConfirmed } = this._contract.interface.events;
-    const web3 = this._txMgr.get('web3')._web3;
-    const topic = web3.utils.keccak256(
-      web3.utils.toHex(LinkConfirmed.signature)
+    const [topic] = this._contract.interface.encodeFilterTopics(
+      'LinkConfirmed',
+      []
```

Comment:
Good example of an ethers built in function that simplifies things a little bit from before.